File: templates/_composition/MVVMLight/Page/ViewModels/ViewModelLocator_postaction.cs
namespace RootNamespace.ViewModels
{
    public class ViewModelLocator
    {
        public ViewModelLocator()
        {
            //^^
            Register<uct.ItemNameViewModel, uct.ItemNamePage>();
        }

        public uct.ItemNameViewModel uct.ItemNameViewModel => ServiceLocator.Current.GetInstance<uct.ItemNameViewModel>();
    }
}

File: templates/Uwp/_composition/MVVMLight/Page.AddViewModelVB/ViewModels/ViewModelLocator_postaction.vb
Namespace ViewModels
    Public Class ViewModelLocator
        Private Sub New()
            '^^
            '{[{
            Register(Of wts.ItemNameViewModel, wts.ItemNamePage)()
            '}]}
        End Sub

'{[{

        Public ReadOnly Property wts.ItemNameViewModel As wts.ItemNameViewModel
            Get
                Return ServiceLocator.Current.GetInstance(Of wts.ItemNameViewModel)()
            End Get
        End Property
'}]}
    End Class
End Namespace
